playerctl : added show what's playing (#3970)
tldr-pages/tldr: pages/linux/betterlockscreen.md

#betterlockscreen

>simple, minimal lock screen

- Lock the screen:

`betterlockscreen --lock`

> -Change lock screen background:

Reviewer: Perhaps this will read better?
```suggestion
- Change the lock screen background:
```

> `betterlockscreen -u path/to/the/image.png`

Reviewer: Perhaps we could use the `{{token_syntax}}` here?
```suggestion
`betterlockscreen -u {{path/to/image.png}}`
```

> -Lock the screen with custom text:

Reviewer: Perhaps this will read better?
```suggestion
- Lock the screen, showing some custom text:
```

> `betterlockscreen -l pixel -t "custom lock screen text"`

Reviewer: Perhaps we could use the `{{token_syntax}}` here?
```suggestion
`betterlockscreen -l pixel -t "{{custom lock screen text}}"`
```

Also, are there any long forms of these arguments? In tldr pages, we prefer the longer form options as it aids clarity.

> -Lock the screeen with custom monitor off timeout:

Reviewer: Perhaps this will read better?
```suggestion
- Lock the screen, with a custom monitor off timeout:
```

> `betterlockscreen --off 5 -l`

Reviewer: Perhaps we could use the `{{token_syntax}}` here?
```suggestion
`betterlockscreen --off {{5}} -l`
```
